NoNicotineAPI/Controllers/TherapistsController.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using MediatR;
 using Microsoft.AspNetCore.Mvc;
 using NoNicotin_Business.Commands;
+using NoNicotin_Business.Queries;
```

Comment:
you changed the name of the project, but the namespace is still NoNicotin. Please fix that so when it imports it say NoNicotine_Business.Queries (for ex.)